include/ze_api.h

```diff
@@ -1187,7 +1282,7 @@ typedef enum _ze_api_version_t
     ZE_API_VERSION_1_11 = ZE_MAKE_VERSION( 1, 11 ),                         ///< version 1.11
     ZE_API_VERSION_1_12 = ZE_MAKE_VERSION( 1, 12 ),                         ///< version 1.12
     ZE_API_VERSION_CURRENT = ZE_MAKE_VERSION( 1, 12 ),                      ///< latest known version
-    ZE_API_VERSION_FORCE_UINT32 = 0x7fffffff
```

Comment:
this is a bug in the spec scripts, you need to manually add the 1.13 version.

Reply:
also, you need to update:
https://github.com/oneapi-src/level-zero-spec/blob/master/scripts/core/driver.yml#L213